biojava-genome/src/main/java/org/biojava/nbio/genome/util/ChromosomeMappingTools.java

```diff
@@ -610,7 +593,8 @@ public static int getCDSLengthForward(List<Integer> exonStarts, List<Integer> ex
         if ( debug)
             s = new StringBuffer();
 
-        int lengthExons = 0;
+        @SuppressWarnings("unused")
+		int lengthExons = 0;
```

Comment:
Why is this variable not used? Just wondering if it would be better to remove it to eliminate confusion.